gobblin-utility/src/main/java/org/apache/gobblin/util/hadoop/TokenUtils.java

```diff
@@ -193,9 +192,12 @@ public static void getHiveToken(final State state, IMetaStoreClient hiveClient,
     try {
       // Fetch and save the default hcat token.
       LOG.info("Fetching default Hive MetaStore token from hive");
-      HiveConf hiveConf = new HiveConf();
 
-      Token<DelegationTokenIdentifier> hcatToken = fetchHcatToken(userToProxy, hiveConf, null, hiveClient);
+      // Fetch and save the default hcat token.
```

Comment:
You can drop the default part - I don't think it conveys a specific meaning. Let me know if otherwise.